src/services/sync/services/UserAccountService.js

```diff
@@ -29,20 +30,25 @@ class UserAccountService {
 	}
 
 	async createAccount(account) {
-		return this.app.service('nest-account-service').save({
-			userId: account.userId,
-			username: account.username.toLowerCase(),
-			systemId: account.systemId,
-			activated: true,
+		return RequestContext.createAsync(this.app.service('nest-orm').em, async () => {
```

Comment:
Initiator of the request is feathers, hence we need the RequestContext created here. Alternatively, to avoid having details of the ORM spill into the legacy stack, we could wrap the nest account related Services and UCs in the nest stack and have the request context setup there. This again would bloat the nest implementations. I'd prefer to keep the responsibilities of setting up the legacy access to nest stack within the legacy stack entirely for the sake of separations of responsibilities.